### PR TITLE
Add various corrections

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -7962,6 +7962,9 @@ extacy->ecstasy
 extarnal->external
 extarnally->externally
 extatic->ecstatic
+extedn->extend
+extedned->extended
+extedns->extends
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Please add corrections for `extender` and `extenders`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -3618,6 +3618,9 @@ comman->command, common,
 commandoes->commandos
 commans->commands
 commansd->commands
+commant->command
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: also `comment`

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Done

**Comment by peternewman** [comment]
> pluralizing [f9d9d05](https://github.com/codespell-project/codespell/commit/f9d9d05db3b8460cf447eef5f92202e342e4fc12) ?

Done

**Pull request merged**